example/lib/main.dart

```diff
@@ -152,6 +152,11 @@ class _MyHomePageState extends State<MyHomePage> {
       pane: NavigationPane(
         selected: index,
         onChanged: (i) => setState(() => index = i),
+        size: NavigationPaneSize(
+          openWidth: MediaQuery.of(context).size.width / 5,
```

Comment:
Why to use this here?

Reply:
Only for the demonstration of the capactity to have a dynamic width with a minimum and a maximum width.